app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -8,36 +8,68 @@ def create_in_provider(manager_id, params)
         connection.api.projects.create!(params)
       end
 
+      refresh(manager)
+      find_by!(:manager_id => manager.id, :manager_ref => project.id)
+    end
+
+    def create_in_provider_queue(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      queue(manager.my_zone, nil, "create_in_provider", [manager_id, params], "Creating #{name}")
```

Comment:
The last argument will be logged when creating the Queue item.  Can you change that to include the `ems_ref` for the project being created? Something like `"Creating #{name} with name #{params['name']}"` maybe...